src/styl/site.styl

```diff
@@ -60,6 +71,12 @@ body
 .line-larger
   line-height: $font-leading-larger
 
+.font-bolder
```

Comment:
I feel like this should just be `bold` rather than `bolder`. We should if possible stick to trad typography terms of weights I think.